packages/medusa/src/loaders/helpers/routing/__fixtures__/server/index.js

```diff
@@ -0,0 +1,171 @@
+import { asValue, createContainer } from "awilix"
+import express from "express"
+import jwt from "jsonwebtoken"
+import { MockManager } from "medusa-test-utils"
+import querystring from "querystring"
+import supertest from "supertest"
+import apiLoader from "../../../../api"
+import featureFlagLoader, { featureFlagRouter } from "../../../../feature-flags"
+import modelsLoader from "../../../../models"
+import passportLoader from "../../../../passport"
+import repositoriesLoader from "../../../../repositories"
+import servicesLoader from "../../../../services"
+import strategiesLoader from "../../../../strategies"
+
+import RoutesLoader from "../.."
+import { config } from "../mocks"
+
+function asArray(resolvers) {
+  return {
+    resolve: (container) =>
+      resolvers.map((resolver) => container.build(resolver)),
+  }
+}
+
+/**
+ * Sets up a test server that injects API Routes using the RoutesLoader
+ *
+ * @param {String} rootDir - The root directory of the project
+ */
+export const createServer = async (rootDir) => {
+  const app = express()
+
+  // const moduleResolutions = registerModules({})
+  // // Load non legacy modules
+  // Object.keys(modules).map((moduleKey) => {
+  //   moduleResolutions[moduleKey] = registerMedusaModule(
+  //     moduleKey,
+  //     ModulesDefinition[moduleKey]
+  //   )[moduleKey]
+  // })
```

Comment:
todo: rm, if not needed

Reply:
It might be needed at some point if we want to test using modules in API Routes, but whenever I include it, it throws an error that registerModules is not exported from modules-sdk. I can't figure out what is causing this as it works fine in the test-request util. I'll delete it for now though as it's not needed for any of the current tests.

Reply:
we are also calling `moduleLoader` in the test helper :)